lebedigital/demonstrator_scripts/dummy_paste_strength_stiffness.py

```diff
@@ -30,8 +30,8 @@ def dummy_paste_strength_stiffness(slag_ratio, phi_paste):
         paste_youngs_modulus_min + (paste_youngs_modulus_max - paste_youngs_modulus_min) * slag_ratio
     )
 
```

Comment:
E also needs to be updated right? similar to the paste_strength i.e., 
paste_youngs_modulus = (
        paste_youngs_modulus_max - (paste_youngs_modulus_max - paste_youngs_modulus_min) * slag_ratio

Reply:
Needs is a strong word :D, this is a dummy script but by implementing the change the trend might be closer to what we would expect in real life.
I made the change as requested.